app/uk/gov/hmrc/cgtpropertydisposals/repositories/returns/DraftReturnsRepository.scala

```diff
@@ -167,8 +166,8 @@ object DefaultDraftReturnsRepository {
   )
 
   object DraftReturnWithCgtReferenceWrapper {
-    implicit val dtf: Format[Instant] = MongoJavatimeFormats.instantFormat
-    implicit val format               = Json.format[DraftReturnWithCgtReferenceWrapper]
+    implicit val dtf: Format[Instant]                                = MongoJavatimeFormats.instantFormat
+    implicit val format: OFormat[DraftReturnWithCgtReferenceWrapper] = Json.format[DraftReturnWithCgtReferenceWrapper]
```

Comment:
FYI: You can avoid doubling the type by putting it on the left hand side and letting the type inference figure out the right hand side.